example/package.json

```diff
@@ -45,6 +45,7 @@
   },
   "dependencies": {
     "@cumulus/api": "1.20.0",
+    "@cumulus/api-client": "1.19.0",
```

Comment:
This should be set to 1.20, however that will cause lint to fail.   Once this has passed code review, I'll manually push out the library and update these references.

Reply:
Yes, correct.     I hadn't anticipated a release occurring before a review, really should just have said <latest>

Reply:
Updated add35c05d - will publish soonish